compiler/daml-extension/BUILD.bazel

```diff
@@ -43,32 +44,15 @@ sh_test(
 # copies them to their intended place, and then bundles the whole node_modules
 # folder as a tarball so the next task, below, can depend on that cached
 # tarball and be fast.
-# Also for some reason on Windows I get "cannot ceate node_modules: file
-# exists", so at this point I'm completely out of trust.
-genrule(
+# XXX: Use rules_nodejs to generate the extension.
+pkg_tar(
```

Comment:
Given that we don’t test the vscode extension if would be great if you could make sure this does not break anything by running `daml-sdk-head --skip-jars` and then `daml studio --replace=always`.

Reply:
I've tested this on an example project. The extension and the scenario runner seem to be working.